docs/iceberg-rest-service.md

```diff
@@ -23,11 +23,19 @@ Builds with Apache Iceberg `1.3.1`. The Apache Iceberg table format version is `
 Builds with Hadoop 2.10.x, there may compatibility issue when accessing Hadoop 3.x clusters.
 :::
 
-## How to start the Gravitino Iceberg REST catalog service
+## Gravitino Iceberg REST catalog service Configurations
 
-Deploy the Gravitino server to the `GRAVITINO_HOME` directory. You can find the configuration options in [`$GRAVITINO_HOME/conf/gravitino.conf`](gravitino-server-config.md).
+Assuming the Gravitino server is deployed in the `GRAVITINO_HOME` directory, you can locate the configuration options in [`$GRAVITINO_HOME/conf/gravitino.conf`](gravitino-server-config.md). There are three types of configurations for the Iceberg REST catalog service:
 
-### Gravitino Iceberg REST catalog service configuration
+1. [**REST Catalog Server Configuration**](#rest-catalog-server-configuration): you can specify the HTTP server properties like host and port.
+
+2. [**Iceberg Catalog Configuration**](#iceberg-catalog-configuration): you have the option to set the specified catalog-backend to either jdbc or hive.
```

Comment:
Either use JDBC or Hive as they are proper names or abbreviations or put in code quotes if they are conjuration values.

Reply:
using `jdbc` and `hive`